api/V1/Group.ts

```diff
@@ -120,6 +121,47 @@ export default function (app: Application, baseUrl: string) {
     })
   );
 
+    app.get(
```

Comment:
These two new apis should probably have some *apidocs comments associated with them.